Triangulation_3/include/CGAL/IO/Triangulation_raw_iostream_3.h

```diff
@@ -0,0 +1,396 @@
+// Copyright (c) 20XX,20XX GeometryFactory
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Ange Clément
+
+
+#ifndef CGAL_TRIANGULATION_RAW_IOSTREAM_3_H
+#define CGAL_TRIANGULATION_RAW_IOSTREAM_3_H
+
+#include <CGAL/license/Triangulation_3.h>
+
+#include <iostream>
+#include <vector>
+
+#include <CGAL/Unique_hash_map.h>
+#include <CGAL/Default.h>
+#include <CGAL/IO/io.h>
+
+namespace CGAL {
+
+template < class GT, class Tds = Default,
+         class Lock_data_structure = Default >
+class Triangulation_3;
+
```

Comment:
Now that this forward-declaration is declared in several headers (here and in `Triangulation_3/include/CGAL/IO/Triangulation_raw_iostream_3.h`), it should be extracted to a dedicated header `Installation/include/CGAL/Triangulation_3_fwd.h`.